### DISCUSSION TITLE
希望加入身份证生成器

### DISCUSSION THREAD

**Discussion by aptx33** [he3-app/he3-feedback#94] [Ideas]
参考页面:
![image](https://user-images.githubusercontent.com/13679006/228449877-8ea3a1a9-855e-4229-94a0-56ddeb399f7c.png)
http://sfz.fatcarter.cn/

**Comment by jingyuexing** [comment]
希望你不是用在灰产之类的

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 Issues，将在后续版本提供支持。

**Reply by jingyuexing** [reply]
你不怕法律风险吗
